query-engine/driver-adapters/js/adapter-pg/package.json

```diff
@@ -18,7 +18,8 @@
   "license": "Apache-2.0",
   "sideEffects": false,
   "dependencies": {
-    "@prisma/driver-adapter-utils": "workspace:*"
+    "@prisma/driver-adapter-utils": "workspace:*",
+    "postgres-array": "^3.0.2"
   },
```

Comment:
Here it's in dependencies, which is good, we should do the same for https://github.com/prisma/prisma-engines/pull/4351/files#r1354763538

Reply:
Fixed! Thank you!